CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.3.0] - 2021-11-19
+### Added
+- Bearer Authentication strategy added
```

Comment:
Since it will alter existing Request constructors if openapi spec already contains bearer auth strategy, I think it should be considered as a breaking change.

Reply:
good point, fixed